package.json

```diff
@@ -73,6 +73,7 @@
     "ember-cli-server-variables": "3.0.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
+    "ember-disable-prototype-extensions": "^1.1.3",
```

Comment:
I'm not sure if I should have gone this way in the addon, but for an app I think [following this guide](https://guides.emberjs.com/release/configuring-ember/disabling-prototype-extensions/) to disable the extensions is probably a better path.

Reply:
:heavy_check_mark: